src/checks_windows.go

```diff
@@ -144,6 +157,12 @@ func serviceUp(serviceName string) (bool, error) {
 	return command("if (!((Get-Service -Name '" + serviceName + "').Status -eq 'Running')) { Throw 'Service is stopped' }")
 }
 
+
+func PasswordChanged(user, date string) (bool, error) {
+	res, err := command("Get-LocalUser " + user + " | select PasswordLastSet | Select-String " + "date\"")
```

Comment:
The strings look weird on this one, pretty sure this will add the literal string `date"`.

Reply:
Isn't that what we want?

Reply:
No, it should be ` + date + "`

Reply:
But for the powershell command i need quotation marks